packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -1452,34 +1694,26 @@ class _AnimatedScaleState extends ImplicitlyAnimatedWidgetState<AnimatedScale> {
 /// ```
 /// {@end-tool}
 ///
+/// {@macro flutter.widgets.AnimatedValue.initialValue}
+///
 /// See also:
 ///
 ///  * [AnimatedScale], for animating the scale of a child.
 ///  * [RotationTransition], an explicitly animated version of this widget, where
 ///    an [Animation] is provided by the caller instead of being built in.
-class AnimatedRotation extends ImplicitlyAnimatedWidget {
+class AnimatedRotation extends AnimatedValue<double> {
   /// Creates a widget that animates its rotation implicitly.
   const AnimatedRotation({
     super.key,
-    this.child,
-    required this.turns,
+    required double turns,
```

Comment:
The document for `turns` is missing, it would be hard to tell what unit it uses.

Reply:
ooh, good point 👍